BibBuddy/app/src/main/res/layout/list_view_item_note.xml

```diff
@@ -16,30 +17,88 @@
 
-        <TextView
-                android:id="@+id/noteName"
+            <ImageButton
+                android:id="@+id/stop_note"
+                android:layout_width="50dp"
+                android:layout_height="50dp"
+                android:layout_centerVertical="true"
+                android:layout_toEndOf="@+id/play_note"
+                android:background="@null"
+                android:contentDescription="@string/stop_button_note"
+                android:scaleType="fitXY"
+                android:src="@drawable/icon_stop" />
+
+            <TextView
+                android:id="@+id/played_time"
                 android:layout_width="wrap_content"
                 android:layout_height="wrap_content"
-                android:layout_marginStart="10dp"
-                android:layout_toEndOf="@+id/noteType"
-                android:text="@string/note"
-                android:textColor="@color/black" />
+                android:layout_below="@+id/seekBar"
+                android:layout_marginStart="60dp"
+                android:text="@string/default_played_timer"
+                android:textSize="12sp"
+                android:translationY="-3dp" />
 
-        <TextView
-                android:id="@+id/noteModDate"
+            <TextView
+                android:id="@+id/total_time"
                 android:layout_width="wrap_content"
                 android:layout_height="wrap_content"
-                android:layout_marginTop="10dp"
-                android:layout_alignParentEnd="true"
-                android:layout_alignParentBottom="true"
-                android:textSize="12sp" />
+                android:layout_below="@+id/seekBar"
+                android:layout_marginStart="258dp"
+                android:textSize="12sp"
+                android:translationY="-3dp" />
+
+            <app.minimize.com.seek_bar_compat.SeekBarCompat
+                android:id="@+id/seekBar"
+                android:layout_width="230dp"
+                android:layout_height="35dp"
+                android:layout_centerVertical="true"
+                android:layout_marginStart="60dp" />
+
+        </RelativeLayout>
+
+        <TextView
+            android:id="@+id/note_mod_date"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_alignParentBottom="true"
+            android:layout_alignParentEnd="true"
+            android:layout_marginTop="10dp"
+            android:textSize="12sp" />
+
+        <TextView
+            android:id="@+id/note_name"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="10dp"
+            android:layout_toEndOf="@+id/note_type"
+            android:text="@string/note"
+            android:textColor="@color/black" />
 
     </RelativeLayout>
-</androidx.cardview.widget.CardView>
+
+</androidx.cardview.widget.CardView>
```

Comment:
In general, this layout seems to cause problems. 

On both my emulator and my smartphone, play/pause button are distorted and too small and too far right (on my smartphone the buttons didnt show up at all). I would suggest using a ConstraintLayout and constraining from right to left to adjust the size of the bar so that the controls are always visible and big enough. You can have a look at the header layout, where i sized the header text based on the width of elements on its right.

Reply:
Also pls add a newline at the end of the file, github says :)

Reply:
Thanks for your bug-report: I replaced the relative voice note layout with a constraint layout and adjusted all child-views as suggested.
I added a new line at the end of the file as well.